scripts/ci/collect-service-logs.sh

```diff
@@ -80,7 +80,7 @@ main() {
         done
     done
 
-    kubectl -n "${namespace}" get events -o wide >"${log_dir}/events.txt"
```

Comment:
whatabout keeping the `-o wide` version as well? (in case there is someone that is attached to that format)

Reply:
Good idea, done.